Script.php

```diff
@@ -737,136 +716,142 @@ function checksize($file)
 //}
 //   
 
-  $count=0;           
-           
-  
-  
+$count=0;           
+        
 
-  //check zise
-    if(!checksize("attachment1"))
-  {
-      echo "20 MB is the maximum file size allowed";
-      return;
-  }
-    if(!checksize("attachment2") && $_FILES["attachment2"]["name"]!="")
-  {
-       echo "20 MB is the maximum file size allowed";
-      return;
-  }
-    if(!checksize("attachment3") && $_FILES["attachment3"]["name"]!="")
-  {
-     echo "20 MB is the maximum file size allowed";
-      return;
-  }
-  
-  
-  
-  
-  
-  if(!is_valid("attachment1"))
-  {
-      echo "Invalid File Type for Attachment 1";
-      return;
-  }
-    if(!is_valid("attachment2") && $_FILES["attachment2"]["name"]!="")
-  {
-      echo "Invalid File Type for Attachment 2";
-      return;
-  }
-    if(!is_valid("attachment3") && $_FILES["attachment3"]["name"]!="")
-  {
-      echo "Invalid File Type for Attachment 3";
-      return;
-  }
-  
-  if($_FILES["attachment1"]["error"] != 0) {
```

Comment:
@Teecloudy 

When is the value for the **"error" key** set?

Hui

Reply:
Please not that we didnt affect these lines. They are the original lines. But if there may be need for any adjustments we will look at it.